insights/parsers/pam.py

```diff
@@ -108,8 +108,8 @@ class PamSSHD(PamDConf):
 """
 
 from collections import namedtuple
-from .. import Parser, get_active_lines, parser
-from ..parsers import unsplit_lines
+from insights import Parser, get_active_lines, parser
+from insights.parsers import unsplit_lines, keyword_search
```

Comment:
Hey folks,

I don't have a *super* strong opinion here, but I generally think that absolute imports are good when you are importing something outside *your* package and relative imports are good when importing things from within your package.  This allows you to rename or relocate a package with the least number of mindless renaming tasks.  The downside is that if you want to extract a subset of a package to another package you may have to fiddle with the imports.